README.md

```diff
@@ -25,50 +21,99 @@ To get started working on Membership you will need to complete the following ste
 2. Work through the setup instructions for [Identity](https://github.com/guardian/identity) and [theguardian.com](https://github.com/guardian/identity)
 3. Start up Membership by running the commands in the [Run](#run) section of this README
 
+<a name="setup"></a>
+## Setup
+
+### Requirements
+
+We require the following to be installed:
+
+- `Java 8`
+- `Node (with npm)`
+- `NGINX`
+
+### Install client-side dependencies
+
+```
+cd frontend/
+npm install
+npm run devSetup
+```
+
+A good way to check everything is setup correctly is to run the tests
+
+```
+npm test
+```
+
+For development you'll also need the following commands:
+
+**Compile assets**
 
-<a name="nginx">
-## NGinx
+```
+npm run compile
+```
+
+**Watch files for changes**
+
+```
+npm run watch
+```
+
+*Note:* We use `grunt` and `bower` behind the scenes but provide [facades for common tasks](https://bocoup.com/weblog/a-facade-for-tooling-with-npm-scripts/) to make setup easier and reduce the number of tools needed for most developers. If you want to work with the full set of build tools install `grunt-cli` and run `grunt --help` to see the list of available tasks.
+
+**Client-side Principles**: See [client-side-principles.md](docs/client-side-principles.md) for high-level client-side principles for Membership.
+
+**Pattern Library**: A library of common patterns used accross the membership site is available at [membership.theguardian.com/patterns](https://membership.theguardian.com/patterns).
+
+### Setup NGINX
 
 To run standalone you can use the default nginx installation as follows:
 
 Install nginx:
 
-Linux: `sudo apt-get install nginx`
-Mac OSX: `brew install nginx`
+- Linux: `sudo apt-get install nginx`
+- Mac OSX: `brew install nginx`
+
+Make sure you have a `sites-enabled/` folder under your nginx home. This varies depending on your setup but should be under:
 
-Make sure you have a sites-enabled folder under your nginx home. This should be
+- Linux: `/etc/nginx/sites-enabled`
+- Mac OSX: `/usr/local/etc/nginx/sites-enabled`
 
-Linux: `/etc/nginx/sites-enabled`
-Mac OSX: `~/Developers/etc/nginx/sites-enabled` or `/usr/local/etc/nginx/`
-Make sure your nginx.conf (found in your nginx home) contains the following line in the `http{...}` block: `include sites-enabled/*`;
+Make sure your `nginx.conf` (found in your nginx home) contains the following line in the `http{...}` block:
 
-Run:
+```
+include sites-enabled/*
+```
 
-`./nginx/setup.sh`
+If you want an automated setup run:
 
+```
+./nginx/setup.sh`
```

Comment:
Don't forget to remove the orphaned backtick since you've got block backticks now. It shows up in the formatted version